doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -451,7 +451,7 @@ struct B {
 b_a_2 = B.A[2];
 
 in Julia:
-type B
+struct B
     A::NTuple{3, CInt}
 end
 b_a_2 = B.A[2]
```

Comment:
I wonder whether this is intentionally referring to different elements in C vs Julia?